ansible/group_vars/all.yml

```diff
@@ -1,105 +1,138 @@
-locale: en_US.UTF-8
-timezone: Europe/Paris
-web_browser: chromium-browser.desktop
+---
+# System configuration and packages - Essential host layer components
+system:
+  locale: en_US.UTF-8
+  timezone: Europe/Paris
+  apt:
+    - apt-transport-https
+    - ca-certificates
+    - gnupg-agent
+    - software-properties-common
+    - util-linux-extra
+    - systemd
+    - hwclock
```

Comment:
Fix this issue regarding this installation package: Error: : Task failed: Module failed: No package matching 'hwclock' is available